api/test/api-common.test.js

```diff
@@ -5262,6 +5262,60 @@ describe('API', () => {
             });
         });
 
+        test('GET /datasetorders (invalid isAppStrict): should return validation error for invalid isAppStrict value', async () => {
+          const { data, status } = await request
+            .get(
+              buildQuery('/datasetorders', {
+                dataset: datasetAddress, // *
```

Comment:
add mandatory chainId
```suggestion
                chainId, // *
                dataset: datasetAddress, // *
```